src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -88,6 +78,15 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
   const absl::flat_hash_map<NodeID, std::shared_ptr<SchedulingResources>>
       &GetClusterResources() const;
 
+  /// Update resources of a node
+  void UpdateResources(const NodeID &node_id,
+                       absl::flat_hash_map<std::string, double> changed_resources,
+                       std::function<void(const Status &)> callback = nullptr);
```

Comment:
Why do we need a callback? It doesn't seem to be used in a glance?

Reply:
I can remove that. Right now it's not useful I think.